include/zephyr/drivers/lora.h

```diff
@@ -15,6 +15,7 @@
  * @{
  */
 
+#include <zephyr/kernel.h>
 #include <zephyr/types.h>
```

Comment:
`zephyr/types.h` could be removed with the addition of `zephyr/kernel.h`.

Reply:
yes, removed